lib/compute_fingerprint.js

```diff
@@ -23,9 +23,13 @@ Fingerprint.prototype.compute = function() {
 
 Fingerprint.prototype.strippedMessage = function(message) {
   var regex = /Function '\S+'/
-    , stripped = message.match(regex);
+    , match = message.match(regex);
 
-  return stripped[0];
+  if (match) {
+    return match[0];
+  } else {
+    return "Anonymous Function";
```

Comment:
How will this behave when there are multiple, complex anonymous functions in one file?